From a13de92989289c92a972d5b97aa383bde3ba6818 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Thomas=20S=C3=A4nger?= <thomas@gecko.space>
Date: Thu, 23 Aug 2018 15:47:48 +0200
Subject: [PATCH] use lua5.3

---
 Makefile.options | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile.options b/Makefile.options
index 2250cc1..0d3fcf3 100644
--- a/Makefile.options
+++ b/Makefile.options
@@ -8,8 +8,8 @@ LD = ld
 SLIB_EXT = so
 
 # Lua binaries
-LUA_BIN = lua
-LUAC_BIN = luac
+LUA_BIN = lua5.3
+LUAC_BIN = luac5.3
 
 # C compiler flags
 # TODO: check alternatives to -D_GNU_SOURCE -fPIC
-- 
2.18.0

